.github/workflows/luajit-integration-ci.yml

```diff
@@ -0,0 +1,79 @@
+name: LuaJIT integration testing
```

Comment:
Can we name this file `luajit-integration.yml`?

Reply:
Fixed.